chainer/links/rnn/n_step_rnn.py

```diff
@@ -50,22 +50,26 @@ class NStepRNNBase(link.ChainList):
 
     """
 
-    def __init__(self, n_layers, in_size, out_size, dropout, **kwargs):
-        if kwargs:
-            argument.check_unexpected_kwargs(
```

Comment:
Fixed in d1d3222

Reply:
ee5e00f

1. I restored `if kwargs:` since it needs in the point of performance.
2. Although I also restored `assert_kwargs_empty`, I still don't understand why `argument.assert_kwargs_empty` is required (I guess `if kwargs:` check if `kwargs` is empty). I'm sorry but could you please give me some explanations for this?